.vscode/tasks.json

```diff
@@ -5,7 +5,9 @@
       "label": "Run tests for current file",
       "type": "shell",
       "command": "script/test_file",
-      "args": ["${relativeFile}"],
+      "args": [
```

Comment:
[Have I not asked you](https://github.com/home-assistant/intents/discussions/399#discussioncomment-7524814) not to merge anything that's not related to Slovenian?

In this particular case, it's not a big issue as you haven't modified anything other than spacing, but it does look like you are not checking your PRs to see if you're also merging stuff that shouldn't be merged.

So, if there's anything edited outside of the `sl` folders (the ones in `responses`, `sentences` and `tests`), please **DONT'T** merge the PR without approval from @synesthesiam.

Reply:
I'm sorry about that, I wasn't aware i changed anything outside os `sl` folder, I'm sure it's a consequence of my not updated fork...anyways it's on my account. Maybe I'm just not up to the task working with github...